addon/components/polaris-choice-list.js

```diff
@@ -58,103 +48,84 @@ export default Component.extend({
    *
    * TODO try to find a nicer API for React's `renderChildren`
    *
-   * @property choices
-   * @public
    * @type {Array}
-   * @default null
+   * @default []
+   * @public
    */
-  choices: null,
+  choices = [];
```

Comment:
Any particular reason for changing this? Also if this has changed from `null` to `[]`, should we make the same change to `selected` for consistency? 🤔

Reply:
changed this because it should be the default....we didn't do this before because of leaking issue before refactored into an ES6

yeah, can make selected too 👍